### ISSUE TITLE
License facet should show the human-readable license labels

### ISSUE THREAD

**Issue by andrewjbtw** [sul-dlss/argo#2758]
**Describe the bug**
The license facet in Argo is showing the URLs for most licenses instead of the human readable labels (except for `pdm` for some reason).

<img width="563" alt="Screen Shot 2021-06-04 at 3 46 37 PM" src="https://user-images.githubusercontent.com/8975583/120869851-33ab9000-c54c-11eb-8c5a-12973921bffc.png">

**User Impact**
It's hard to parse out which one means which.

**To Reproduce**

There's a collection in stage that is the combination of all licenses available in Argo and all licenses available in H2:
https://argo-stage.stanford.edu/catalog?f%5Bis_member_of_collection_ssim%5D%5B%5D=info%3Afedora%2Fdruid%3Aqz475ck8409

**Expected behavior**
The licenses should show the human-readable names. It would be preferable to distinguish versions as well, so we can know when we have Creative Commons 3.0 vs. 4.0, for instance.

**Comment by jcoyne** [comment]
@andrewjbtw can you point me to where we find the human readable names you want to see for each possible license?

**Comment by andrewjbtw** [comment]
I don't know where they're stored. Production has been showing them this way since I started using Argo (I'm not sure why stage is doing something different right now):

![Screen Shot 2021-06-07 at 9 18 34 AM](https://user-images.githubusercontent.com/8975583/121054649-6b475180-c771-11eb-87bb-0febeb2f20c5.png)

Maybe the current indexing is reading them directly out of the rightsMetadata? 

I think the SPDX identifier list is pretty clear and compact enough to fit into the facet, so I think we should use that.

**Comment by jcoyne** [comment]
@andrewjbtw the indexer currently prefers the new `<license>` tag and falls back to the `<machine>` tag values.

**Comment by jcoyne** [comment]
@andrewjbtw this code is now live and I've kicked off reindexing jobs for all of the URI facet values.
